docs/index.md

```diff
@@ -18,6 +18,8 @@
 
 [Mesa] is an Apache2 licensed agent-based modeling (or ABM) framework in Python.
 
+For the original conference paper click here: http://conference.scipy.org.s3-website-us-east-1.amazonaws.com/proceedings/scipy2015/jacqueline_kazil.html
```

Comment:
Currently is doesn't render nicely, what do you think of this?
```suggestion
The original conference paper is [available here](http://conference.scipy.org.s3-website-us-east-1.amazonaws.com/proceedings/scipy2015/jacqueline_kazil.html).
```

Reply:
Okay will do that anything else that catches your mind please share thanks.